deps/oob-1.0.2/jsonutil.c

```diff
@@ -404,7 +530,84 @@ void setoobConnectioninfo(const char* host,
     copyString("https", connectionInfo.protocol, 8);
   else
     copyString("http", connectionInfo.protocol, 8);
+}
 
+void setoobdnsinfo(const char *key, const char *val) {
+    char* caps = malloc(strlen(key));
+    upper(caps, key);
+
+    if (!strcmp(caps, "DESCRIPTION")) {
```

Comment:
Why are we doing this ? Can't we just store this as an array of pairs ? And pass in an array of pairs ?

Reply:
Same issue that these "pairs" don't exist in C so I'd have to create a struct regardless. The best we can do is change it to Viswa's suggestion.

Reply:
By pair  I simply mean const char* kvpair[2]